### PR TITLE
Allow people to text caviar driver

### PR DESCRIPTION
**Pull request by timminkov**
@ledwards checkitout

### CODE REVIEW

File: server.rb [hunk 1]
```diff
@@ -4,10 +4,17 @@
...
 require File.expand_path('../lib/services/slack', __FILE__)
 
+account_sid = ENV['TWILIO_ACCOUNT_SID']
+auth_token = ENV['TWILIO_AUTH_TOKEN']
+
+@client = Twilio::REST::Client.new account_sid, auth_token
+
 class Server < Sinatra::Application
+
+  SIX_HOURS_AGO = 60 * 60 * 6
```
Comment (ledwards) [inline server.rb]: Could do `6.hours.ago`

File: server.rb [hunk 2]
```diff
@@ -27,9 +34,25 @@ def slack_client
     "OK"
   end
 
+  post '/text' do
+    unless @from && (Time.now - @time) > SIX_HOURS
```
Comment (ledwards) [inline server.rb:38]: I think you mean `SIX_HOURS_AGO`

File: server.rb [hunk 3]
```diff
@@ -39,7 +62,10 @@ def slack_client
       r.Say "Hello, Caviar driver. I am an automated answering system, but someone will call you back at this number shortly. You can also send text messages to this phone number and someone will read them."
     }.text
 
-    slack_client.alert("The Caviar driver called the contact number. Please return their call at #{params[:From]}. Thanks!")
+    @from = params[:From]
+    @time = Time.now
+
+    slack_client.alert("The Caviar driver called the contact number. Please return their call at #{params[:From]}. You can also send them a text by using @caviar. Thanks!")
```
Comment (ledwards) [inline server.rb]: want to use `@from` here?

File: server.rb (hunk 1 above)
Comment (timminkov) [inline server.rb]: Can't without adding activesupport

File: server.rb (hunk 2 above)
Comment (timminkov) [inline server.rb:38]: Nope, that's six hours, not six hours ago

Comment (ledwards) [inline server.rb:38]: Then maybe the constant up top should be called `SIX_HOURS`?

Comment (timminkov) [inline server.rb:38]: Ahhhhh oops

**Comment by timminkov** [comment]
@ledwards updated

**Comment by ledwards** [comment]
Looks good! 👍 Merging and deploying

**Pull request merged**